resources/com.sap.piper/pipeline/cloudSdkStageDefaults.yml

```diff
@@ -55,6 +55,10 @@ stages:
           - 'appUrls'
   performanceTests:
     stepConditions:
+      gatlingExecuteTests:
+        filePatternFromConfig: 'testModule'
```

Comment:
I would rather name it `pomFile` or something similar to be consistent to other steps.

Reply:
pomPath

Reply:
Ok.